src/realm_dart.cpp

```diff
@@ -53,9 +53,12 @@ RLM_API void realm_dart_initializeDartApiDL(void* data) {
     Dart_InitializeApiDL(data);
 }
 
+using namespace realm;
+using namespace realm::c_api;
```

Comment:
Do we need these statements now in this fail?

Reply:
No, will remove